docs/en/latest/plugins/cors.md

```diff
@@ -35,15 +35,15 @@ The `cors` Plugins lets you enable [CORS](https://developer.mozilla.org/en-US/do
 ### CORS attributes
 
 | Name                      | Type    | Required | Default | Description                                                                                                                                                                                                                                                                                                                                                                                        |
-|---------------------------|---------|----------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
please avoid making unrelated changes like addition/removal of hypens/whitespaces. You can revert them to as they were before.

Reply:
ok